src/app/portlets/content-types/fields/content-type-field-dragabble-item/content-type-field-dragabble-item.component.html

```diff
@@ -0,0 +1,6 @@
+
+<span>{{field.name}}</span>
+<button pButton iconButton flat 
+        icon="fa-remove"
+        (click)="removeField()">
```

Comment:
I think you don't need the method you can do the emit directly here in the template.